packages/client/src/components/Sidebar/constants.ts

```diff
@@ -141,7 +139,7 @@ export const sidebarList: SidebarList = [
     icon: iconApps,
     enabledIn: [FEATURES.CE, FEATURES.EE],
     stage: 'beta',
-    hidden: !window.enableApp,
+    flag: 'enableApp',
```

Comment:
I wonder if we can remove `flag` for App menu item since it's used by App only

Reply:
Yep, I think `flag` can be removed, also I need adjust `Sidebar` component line 53 like below:

```js
if (item.title === 'Apps' && !enableApp) {
	return false;
}
```

How do you think?